test_build_bundle_image.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+source build_bundle_image.sh --test
+source _test_common.sh
+
+function main {
+	set_up
+
+	test_set_parent_image_2025_q1_0
+	test_set_parent_image_2024_q3_0
+	test_set_parent_image_2024_q2_0
+	test_set_parent_image_7_4_13_u125
+	test_set_parent_image_7_4_3_120_ga120
+	test_set_parent_image_7_3_10_u36
+	test_set_parent_image_7_2_10_8
+
+	tear_down
+}
+
+function set_dockerfile {
+	echo -e "FROM --platform=amd64 liferay/${1}:latest AS liferay-${1}\n" > "${4}"
+	echo -e "FROM liferay-${2}\n" >> "${4}"
+	echo "${3}" >> "${4}"
+}
+
+function set_up {
+	export TEMP_DIR="${PWD}"
+}
+
+function tear_down {
+	unset TEMP_DIR
+}
+
+function test_set_parent_image_2025_q1_0 {
+	LIFERAY_DOCKER_RELEASE_VERSION=2025.q1.0
```

Comment:
@natocesarrego, should we do like this `LIFERAY_DOCKER_RELEASE_VERSION="2025.q1.0"`?

Reply:
I believe it makes sense. :slightly_smiling_face: Will do!